fish_prompt.fish

```diff
@@ -285,6 +285,17 @@ function __bobthefish_git_dirty_verbose -S -d 'Print a more verbose dirty state
     echo "$changes " | string replace -r '(\+0/(-0)?|/-0)' ''
 end
 
+function __bobthefish_git_stashed -S -d 'Print the stashed state for the current branch'
+    set -l stashes (command git rev-list --walk-reflogs --count refs/stash 2>/dev/null)
```

Comment:
Counting stashes for users who haven’t explicitly asked for it is probably too much of a performance hit to take. Have you profiled this against the current stash flag implementation?

Reply:
I made a little tweak in 69516c8. Now stashes are counted only if the variable is set to `yes`.